src/entities/user.entity.ts

```diff
@@ -0,0 +1,17 @@
+import { Entity, Column, PrimaryGeneratedColumn, Unique } from 'typeorm';
+
+@Entity('users')
+export class User {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
```

Comment:
While doing validation, maybe we should add column length here also, So that the validation min-max rules are synced with DB column length.